### PR TITLE
Crit Rework

### PR DESCRIPTION
**Pull request by Fox-McCloud**
This is basically Goon's crit system, so all credit goes to Cogwerks, who was the primary designer around this.

In any event, I've wanted to do this for a long time. Our crit system isn't terribly fun or engaging. It's just kinda a case of you become really slow, then you lapse into hard crit and stare at a black screen until death. It's not terribly interesting for doctors, either, as treating someone who comes in tends to just be "treat underlying damage and send them on their way".

Either case, this crit system will completely turn that on its head. When you hit `0` health, your vision will get blurry, you'll get lots of scary message, and you'll start rolling for a chance to lapse into shock. Your breathing will also become irregular. Shock is treated with saline solution.

If shock is left untreated, you will then acquire the state of cardiac emergency. Here, it becomes even more difficult to breathe, and, if left untreated, your heart will stop. You can treat cardiac emergencies with either epinephrine or atropine. 

A stopped heart requires a defib. Not shockingly (no pun intended), defibs are no longer tools for revival; they exist purely to jumpstart hearts. Stopped hearts behave like heart-attacks (paralysis, rapid O2 damage, and brain damage).

On that note, how do you die, you might ask? Well, there's no set point, in terms of overall damage. If you have 200 total damage, you're going to black out until you die/are saved, but otherwise, the point at which you die will either be  when brain dies or randomly, weighted towards how much damage you currently have accumulated.

By now, there's a few of you who are wondering how this impacts other mobs, or even some species. Things like xenos, simple animals, and the likes all die like they did before. They accumulate so much damage, they lapse into hard crit, accumulate O2, then die (or in the case of simple mobs; they just die).

Some species do not work well with this new system, at all--as it's heavily dependent on organs and breathing. As such, some species will utilize the old crit system, lest they become unkillable. Currently Slimes, IPCs, Plasmamen and Diona die at specific thresholds (Antag/special species like shadow/shadowlings, skeletons, golems, and abductosr all die at thresholds as well). Of course, some of these species could lose their trait of no breathing and possess a heart, thus allowing them to be put on this new system, but that's kinda taking variety away...and it would really only work for Diona and Slimes

Other changes
- Refactored the defib so that borg defibs are a subtype as opposed to horrifyingly snowflaked object

:cl: Fox McCloud
add: Crit system completely rehashed; treating crit now requires multiple drug types and is strongly dependent on brain damage for death
del: Defibs no longer revive people
tweak: Corazone recipe removed (reagent remains)
/:cl:

### CODE REVIEW

**Comment by TDSSS** [comment]
While I am not opposed to this change in principle, is now really the time to heap more changes/relearning on medbay when we have like a bazillion revival/medbay change PRs coming?

Also, does this remove the use of painkillers like hydrocodone, morphine, etc. of preventing pain crit, or do they work to prevent the new shock state?

**Comment by Fox-McCloud** [comment]
Paincrit is done. Paincrit was part of the old soft crit system.

**Comment by MrMagolor** [comment]
You would be able to detect shock/failure/arrest with a health analyzer, right?

**Comment by Fox-McCloud** [comment]
Yes.

**Comment by Spartan6** [comment]
I’m a fan for trying it out.

**Comment by TDSSS** [comment]
I know paincrit's done. I'm asking if the chems used to treat paincrit will be given a new use to compensate or have just been nerfed.

**Comment by Shadeykins** [comment]
Neat!

**Comment by DesolateG** [comment]
Can you change the description from a stopped heart to Cardiac Arythmia?

**Comment by granodd** [comment]
Vaguely reminds me of Bay's Brainmed system where the life of the patient is ultimately tied to the status of the brain. Is there any way of bringing someone back from braindeath with this system outside of cloning?

**Comment by Fox-McCloud** [comment]
Strange reagent will bring them back, but they're likely to just die immediately after. Reviving people with strange reagent is going to be particularly difficult with this new system. Very involved, and very touch and go.

**Comment by Bookdrizzt** [comment]
I don’t have a problem with the crit system, but if literally any other changes besides this go through to SR, it’s going to be a huge fuck you to vox and slime people. No defibs, no cloning, and SR is either removed or really hard to get and scarce. I’m still not a fan of this, because defibs have already been nerfed down to two minutes. That’s actually realistic in terms of how you can restart the stopped heart of a dead person, and I don’t see how removing their reviving functionality entirely is going to help anything.

**Comment by KasparoVy** [comment]
Holy fuck this is gonna be sick
I'm ready to die, Voxing it up as always

**Comment by Mitchs98** [comment]
Agreeing with Book. This is actually a reasonable PR to give a nice trial run too. Provided no changes to SR is made, given it will now be more necessary than ever and use up pills faster while reviving people. This is actually quite interesting, I'd be willing to see it in action and it could prove a worth while change; again so long as SR isn't touched. At-least...not touching SR at first if you people still insist on 'nerfing' it.

**Comment by Fox-McCloud** [comment]
Marked as DNM because this will require a bit of livetesting first for potential bugs.

**Comment by DarkPyrolord** [comment]
As someone who started SS13 on Goon and has been playing a bit of Goon as of late, I will say this is one thing I wish more servers would adopt and glad there is a PR for it here as crit in most cases is just bland to say the least, can't wait to see it tested!

File: code/datums/diseases/critical.dm [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/datum/disease/shock
```
Comment (Aurorablade) [inline code/datums/diseases/critical.dm]: Could use buff/debuff framework for this? I mean we do have it.

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1651,6 +1651,9 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 	..()
 
 /mob/living/carbon/human/proc/do_cpr(mob/living/carbon/human/H)
+	if(H == src)
+		to_chat(src, "<span class='warning'>You cannot perform CPR on yourself!</span>")
```
Comment (Aurorablade) [inline code/modules/mob/living/carbon/human/human.dm]: WELL WHY THE HELL NOT?! THIS IS BS SHITAINER REEEEEEEEEEEEEEEE

File: code/datums/diseases/critical.dm (hunk 1 above)
Comment (Fox-McCloud) [reply code/datums/diseases/critical.dm]: the system works better as a disease, as it will show up on medical HUDs. Also just the natural of progression/removal and having it died to reagents and based on stages? It's a perfect shoe-in---I'd have to re-create effectively the same systems for a status effect.

File: code/modules/mob/living/carbon/human/human.dm (hunk 2 above)
Comment (Mitchs98) [reply code/modules/mob/living/carbon/human/human.dm]: Agreed. Literally unplayable. Changed my mind. Do not merge ever.

**Comment by SpinningSalami** [comment]
If going into shock is a disease will having a triple healing virus affect going into shock in any way since 3 is the max amount of diseases you can have?

**Comment by Couls** [comment]
Does this play nice with no breathing genes and viruses and implants? I would assume it bypasses these things

**Comment by Fox-McCloud** [comment]
>If going into shock is a disease will having a triple healing virus affect going into shock in any way since 3 is the max amount of diseases you can have?

That limits is only for advanced viruses.

>Does this play nice with no breathing genes and viruses and implants? I would assume it bypasses these things

Species that have no breathing by default are going to die at a threshold, so they use the old system.

Having the breathless gene from genetics means you're going to still take brute damage from being in crit, still acquire shock, still acquire heart problems, and still roll for brain damage. You'll be a bit more durable, but still killable.

**Comment by Rurik123** [comment]
Sorry if this has been touched on; I may have missed it.

After reaching 200 damage and blacking out, is the succumb command still an option to ghost safely?

File: code/modules/mob/living/carbon/human/species/abductor.dm [hunk 3]
```diff
@@ -16,7 +15,7 @@
 		)
 
 	species_traits = list(NO_BLOOD, NO_BREATHE, VIRUSIMMUNE, NOGUNS, NO_EXAMINE)
-
+	dies_at_threshold = TRUE
```
Comment (datlo) [inline code/modules/mob/living/carbon/human/species/abductor.dm]: I understand the change for other species, but why remove the abductor's heart and not have them use this new system? There's no difference medically between an abductor and a human as far as I know. Is this for balance reasons?

File: code/modules/mob/living/carbon/human/species/vox.dm [hunk 4]
```diff
@@ -185,7 +186,6 @@
 	icon_template = 'icons/mob/human_races/r_armalis.dmi'
 
 	has_organ = list(
-		"heart" =    /obj/item/organ/internal/heart,
```
Comment (datlo) [inline code/modules/mob/living/carbon/human/species/vox.dm]: Perhaps a subtype/new type of heart should be used for species that still uses the old crit system. It's strange that they don't have a heart, period.

**Comment by Ralta** [comment]
This. THIS is the change to dying and medbay we need. I don't think it's necessarily the _only_ change, but this is a huge step forward. Making Saline, Epinephrine and Atropine useful beyond their basic functions? Beautiful.

If we can just get a bit of a nerf to SR and cloning, we'll be in a much better position, imo. Nice job Cogwerks for design, and Fox for implementation. 

:+1:

File: code/modules/mob/living/carbon/human/human.dm (hunk 2 above)
Comment (KasparoVy) [reply code/modules/mob/living/carbon/human/human.dm]: What do you mean I can't punch myself in the chest or run into a wall with enough force to compress my chest and kickstart my heart

**Comment by alex-gh** [comment]
> A stopped heart requires a defib.

Are shocked airlocks/grilles also fine as a ghetto method?

**Comment by UmeFuu** [comment]
I like the idea of the new shock sistem, but I'm not sure I really like all the things that are being done to put more weight on death. I hope someone gives the chems for the old shock sistem (hydrocodone, morphine, etc) a new use for this one, instead of just leaving them useless. 
I know there's already an cpr mechanic, but it sucks how little time you have to use it (As long as I know, It only stopped O2 loss when someone was on crit) it would be interesting to fix it to fit the new sistem and make it useful as a way to keep someone alive until help arrives. Something like, when someone hits the point of shock you can give mouth-to-mouth breathing to prevent the oxy loss, or when their heart stops you'll need to do the whole cpr (breath and  chest compressions), just to keep them alive (Hopefully stopping or reducing the amount of oxy loss and brain damage) and give the doctors more time to work. But I'm just being an idea guy here.
If we're reworking things to make death more impactful, let's add some interesting mechanics to balance it.

**Comment by Mitchs98** [comment]
Actually just thought about this last night. But I'm against the defib change and can't support the PR with it in? Why. It's very much needed during surgery to keep patients alive during critical operations and needed when SRing slimes and vox with toxin damage, It can be very hard if not impossible to revive a slime or vox with heavy toxin as it is. The defib helps immensely with it and makes it possible. I can't support something that's basically indirectly trying to support permadeath for the species if they take certain damage types. The same thing would happen if they took massive brain trauma from the SR happening to them, as it usually requires multiple surgeries and defibs to fix it.

Can't honestly support such a thing, as it kinda feels like a stealthy way to partially achieve the permadeath you all seem to so surely want to for them.

File: code/modules/mob/living/carbon/human/species/abductor.dm (hunk 3 above)
Comment (Fox-McCloud) [reply code/modules/mob/living/carbon/human/species/abductor.dm]: They also have `NO_BLOOD`, which means the heart really has no reason to be there. It doesn't do anything; the flag prevents having a heart attack, which is a major component of the new crit system. They also have `NO_BREATHE` which is also problematic.

**Comment by WingedYordle** [comment]
Are we able to add crawling to crits? Just be able to move to find help before blacking out

**Comment by Fox-McCloud** [comment]
>After reaching 200 damage and blacking out, is the succumb command still an option to ghost safely?

Yup. Suiciding and succumbing both work fine with this new system.

**Comment by Fox-McCloud** [comment]
>Are shocked airlocks/grilles also fine as a ghetto method?

Yup.

**Comment by EvadableMoxie** [comment]
This is exactly the type of change to the medical system I like to see. Makes things more dynamic and more fun and skill based for doctors.

**Comment by KasparoVy** [comment]
> Are we able to add crawling to crits? Just be able to move to find help before blacking out

I hope so. Crawling ftw.
Hearing this reminds me of all those yorpyorpyorp and plumphelmetpunk vids where dudes on Goon would straight up slide around sideways

One of the times I've laughed the hardest, first seeing that

**Comment by Agameofscones** [comment]
Corazone should be added as a stop gap along with Atropine and Epinephrine to prevent or delay cardiac arrest/emergencies. It's a criminally under used chemical that is specific to the heart alone.

**Comment by Fox-McCloud** [comment]
I think this is as good as it gets, aside from live-testing.

Time to move onto that stage.

**Comment by Mitchs98** [comment]
Still not too big of a fan of defibs not working as they do...but that can be changed if proven too detrimental during testing.

**Comment by EvadableMoxie** [comment]
An easy and logical change should be to make it so defibs still work as normal for races not covered by the new critical system.  Otherwise it's an unwarranted and unnecessary nerf.

**Comment by Mitchs98** [comment]
> An easy and logical change should be to make it so defibs still work as normal for races not covered by the new critical system. Otherwise it's an unwarranted and unnecessary nerf.

Basically this.

**Comment by Nicaragua1** [comment]
Cant way to see this system implemented

**Comment by Fox-McCloud** [comment]
Just a few things noted on live-testing. 

Cloning is a bit wonky and I suspect carries a rare risk of individual just straight up dying, though very rare.

Plasmamen should probably be cloneable and not ignite until after being ejected.

**Comment by Mitchs98** [comment]
What of mine and Moxie's concerns regarding defib use..?

**Comment by Couls** [comment]
After the live test I gotta admit I’m all for this system but I agree with the concerns about heartless races and being undefibbable hopefully there’s a solution to heartless races included before this is merged( perhaps a unique crit for the other four races as well)

**Comment by Fox-McCloud** [comment]
Alright, cloning should be stable now for those who go through it.

Also fixed a long-standing issue with Plasmamen burning to death in a pod.

**Comment by Mitchs98** [comment]
Going to be posting a few comments here on the git from the crit guide thread, as I'm unsure if they have a git account or not;

>  Aslan4486;
> In my opinion this system is not very good at all. Instead of trying to make death more impactful, it seems to just lean toward encouraging people to throw everyone in the miracle cloner and boom they are back to good health. The issue lies with cloning, and the medical system before worked perfectly fine. Having to defib people and give them medications and do surgery on them is quite a lot more fun than this new system, which just encourages people to clone people. If cloning was removed all together with SR and defibs being the way to revive people, then death would be more impactful there. Instead of being able to clone people and have them back up and ready to go, you have to do surgery to fix all that is wrong, and the longer someone is dead the harder they are to fix due to the way SR works, which is more fun for medical staff to do. This new system doesn't seem to make death any more impactful except for slimes and vox and such, and just makes medical less fun to play, as throwing someone in the cloner and going "Yep you're good" isn't very fun.

> Tomite;
> 
> This update is flawed in a few ways most of which come from the idea that death needs to be harder to fix and that injuries need to be more life threatening, They don't. Medical does not need to be so complicated that new players wont even try it. The first problem is the silencing of patient's with any amount of o2 damage over ten, This makes initial diagnosis more annoying and doesn't help a medic or doctor fix them without first using a scanner as sometimes its a simple I need a patch or I need blood thus making each patient take longer to get treatments. Second is changing the defibrillator, The defibrillator already had a timer that worked more in line with brain death than you think as any period of time after the defibs window meant they needed cloning or more advanced treatments, The defibs already helped with dealing with heart problem when they were rarer treatments but again that was a more advanced treatment and wasn't nearly as common. Third is the idea that making sr the main way of reviving is an incredibly bad idea, Never mind that it gibs people if they are over a certain damage threshold never mind that it causes genetic damage as someone is revived by it and never mind the fact that some chemists flat out do not make it, This does not solve the core issues with dealing with these patients which would be their cause of death not that they died. When dealing with anything that is not a slime person or vox they will just be tossed in the cloner when things are taken away. Fourth is that the new shock system makes it more likely that patients will arrive dead instead of alive and treatable, Lets run through a hypothetical scenario lets say that you are minding yourself when Grey Tide walks over and just clubs you to your critical state first you walk it off and start heading into medbay then you go into shock and now you are having issues walking finally you stand up but oh no now you are entering cardiac failure and you can not speak because you start taking oxygen damage ignoring that somehow your lungs were not damaged and the only issue is your heart now, You can't speak so nobody can assist you, You cant move and you now have something that most people will ignore when they go for treatments and now congrats a simple problem is now made worse by changing something that honestly did not need a change. This new crit system seems to ignore that we have organ damage already if your lungs take enough damage they collapse if your brain gets damaged you start losing cognitive functions and if you heart takes damage then surprise you can have heart attacks that need treatment. This update seems less like an attempt to make people be more like doctors and more like coroners taking bodies to cloning and then the morgue rather than apply treatments, There is a point in game design where realism has to take a back seat to fun and how people have fun.﻿

Annd now my own post to the thread which sums up my opinion much better than previously;

Echoing everything said above. Medical was fine as it was, I'll always change this. Maybe change a few things like cloning, the sleeper change wasn't that bad even and could even be seen as a buff, or change chemistry up a bit. But this new system, the minor experiences I've had with it, are just the above. It wouldn't be AS BAD if defibs actually revived too. But defibs not reviving? No. It's not really acceptable at all. I know Dave and some others are probably reading this thread going 'Waa, you can't be revived anymore if you do dumb things!! or etc.' But that's not the point here. This system entirely goes against what the majority like about the server, what they want. Not the few minority HighRPers that for some inexplicable reason suddenly want to start making death insane/make permadeath a reality. 

I agree, maybe some more variety could be good. But that could be done by adding in some new systems, new diseases, new surgeries. Just not this. The literal only thing this is going to cause is people relying on cloning like an easy bake oven and transplanting vox and slimes into monkies more frequently because it's "Easier and faster to do this now anyways.". 

I like Paradise because of how it is. A fun chaotic medium RP server that's decently lax on rules and doesn't have overtly punishing mechanics. That's what Paradise is to me and a lot of others. That's actually sort of what it says on the tin 'A Perfect mix of RP and Action'. 

You can say people get to comfortable with the meta, people expect things to always go their way, people are entitled. It has nothing to do with being entitled. Or any of that. It has something to do with what people expect from Paradise, the certain little niche it fills for everyone. There is a REASON we're one of the top ranked servers. That's because most of the people playing here like how it is. They're able to join in, chill with some friends IC, RP, do some zany maint project, and occasionally get the big succ from a changeling.

Changes like this will only majorly detract from the fun. Especially from those of us like myself who enjoy playing medbay to treat people, who realizes it does actually take some semblance of a brain to treat people beyond 'Slap 'em in cryo and go.'.

Call me someone in favor of lowRP or what have you, but I'm not. And this change isn't going to do anything for RP. If anything it's going to make Doctors care less about doing their job and just shove them in cloning/get more tubes built, then sit it and forget it more than they already do.

As far as the whole defib not reviving thing it's actually quite the massive nerf to vox and slime people, as it's very much a key component when reviving them as they often have hilariously high ox/tox damage and defib waves are the only way to treat it. That or so much brain damage they immediately die on revival.

The only things I truly see with this PR is trying to make permadeath a much more likely scenario for those two as old staff wish it to be as "It used to be this way. It was intended to be this way and never should of change.", and punishing competent medical staff or anyone unlucky to get critically wounded/die by...anything. Sure you're more durable, but that really means nothing if you can't cry out for help or get to help. That's a fallacy of an argument, as is Dave's remarks of "You have more time to defib people now.". Having more time to defib people is meaningless if it doesn't actually really help all that much. At all.

TL;DR: This PR isn't really fit for Paradise, it's playstyle, or it's community. And the fact that so many are against it in it's current state(or some even at all) should be proof enough for that. You might not are always supposed to listen to feedback from others, however, when you're overhauling such a major mechanic and the majority have a problem with it? Or even quite a few people have a problem with it? Something is wrong, you need to sit back and listen, take it into account, and realize this may not be a good idea without heavy changes or at all.

**Comment by Fox-McCloud** [comment]
Ensured less confusion for health HUDs, fixed missing missing hearts causing heart attacks while dead (what), and altered food poisoning to not be quit so snowflakey.

**Comment by Mitchs98** [comment]
A response to feedback that's not in favor of the PR beyond a simple downvote would be quite nice.

**Comment by MrMagolor** [comment]
>You might not are always supposed to listen to feedback from others, however, when you're overhauling such a major mechanic and the majority have a problem with it

@Mitchs98 You say that, but then why are the majority of the reactions on the GitHub, the reasonable place to approve or disapprove of a PR, positive?

**Comment by Mitchs98** [comment]
The majority of the community does not have Github. That simple.

**Comment by runemeds** [comment]
With the ambitious recent PRs to medical systems, I vote that it's also up to Fox to be responsible for the necessary medical guide overhaul, and have it finished before this starts getting live tested. With how much has happened the medical roles will require relearning from every facet. It is such a big and complex change that you will find medbay is going to take a long time to adjust to it without concise and well communicated guidance.

**Comment by Fox-McCloud** [comment]
Fixed up Drask healing too much.

Also unified damage checking procs and damage setting procs to avoid potential issues.

`BREATHLESS` and `NO_BREATHE` are now checked for on `adjustOxyLoss` and `SetLoseBreath` instantly instead of relying on a species life loop to reset them to `0`.

**Comment by Fox-McCloud** [comment]
pulled back cryo tubes a bit to further prevent them from being cure-alls.

adjusted the critical ailments so having atropine and epinephrine, both, is of advantage.

survival packs no longer start with auto-injectors (centcomm, nukie, and ERT are exceptions, of course)

CMO's hypospray starts with mitocholide as opposed to omnizine.

Will probably be adjusting the medical vendors tomorrow to make a few crucial changes so medbay isn't neutered if there's no chemist on duty.

**Comment by Mitchs98** [comment]
Definitely 100% not a fan of you not starting with an epi-pen. The change to the hypo is fine, doctors can use donks. But. Not starter with an autoinjector in a system that seriously needs them? Not whatsoever. It's a survival box. It should stay as one. One epi won't be detrimental, if anything it'll rarely be useful at all on yourself and more use on others.

**Comment by Ralta** [comment]
One of the reasons I was supportive of this because it makes the user more aware of when to use an epi pen, instead of possibly wasting it (or throwing it away). Removing them from the kits is a huge mistake, in my opinion. Crew should have a chance to get themselves some help; removing the epi pens prevents that.

**Comment by Dinarzad** [comment]
I agree with concerns over the epi-pen removal from the starter box.
The other changes all seem fine to me, the medi-vend proposed change could fix a lot of complaints depending on what you decide to do. The SR buffs seem very nice too.
Bbbbut, the emergency box having an epi-pen seems pretty important to me. The entire purpose of it was an epi-pen to keep you going for just a few seconds longer if you hit crit, in this system it's a way to keep you out of shock for a few vital seconds while you either try to get help, or scream for someone to come get you.  I highly doubt it'll be too powerful to spawn with (Unless I sorely misunderstand the system, which, I mean I'm not the sharpest lightbulb in the candy shack, so.)

The only other suggestion I could make would be if it's possible to put a defib unit in the paramedic's locker, if he does not have one already. In this new crit system, he's gonna need one to help keep patients alive/stable until they can get to Medbay proper.

**Comment by Fox-McCloud** [comment]
adjusted the vendor start supplies:

img

advanced medkit replaeced with neurological damage kit; individuals who started with advanced medkit now start with a regular medkit, instead.

**Comment by Mitchs98** [comment]
As with the epi pen change, I'm against the removal of the burn/brute trauma kits as well as the ointment from the emergency vendors. Everything else is alright and very much needed. But otherwise? No. It makes no sense to remove the advanced kits fullstop either in favor of 'neuro kits' that are literally mannitol pills anyone can make easily. Once again this is just going further and further from 'make death impactful' to full on nerfing medical into the ground.

I'm definitely against the idea of giving a slime a heart and making them breathe, that's the main upsides to the race really. Not having to worry about organs or breathing. Full on changing races like this to make them work with a convoluted system isn't really the way to go about it in my opinion. 

Sure now the medivends have the chemicals they need to treat shock and stuff effectively, but adding that in favor of _major_ nerfs? I'd rather keep the advanced kits and have to worry about chemistry doing their job. Especially if there's no way to order more of them.

**Comment by EvadableMoxie** [comment]
I thought the reason we nerfed sleepers was to remove the specialist meds to create reliance on chemistry.  Now they're just going to be in vendors? Then what was the point? 

I thought the purpose of the new crit system was to create a situation where we have more people arriving in medical in crit and less being outright dead.  But we're removing Epi-pens, the main stabilization method from starter boxes?

I always felt the general consensus was races were too similar in terms of mechanics, and that the 'reskinned human' thing was a bad thing.  Now we're going to make everyone have a heart and breathe?

I have no idea what's even going on anymore.

File: code/modules/surgery/organs/lungs.dm [hunk 5]
```diff
@@ -19,19 +19,11 @@
 	var/safe_toxins_max = 0.05
 	var/SA_para_min = 1 //Sleeping agent
 	var/SA_sleep_min = 5 //Sleeping agent
-	var/gas_toxicity_multiplier = 100
```
Comment (Fox-McCloud) [inline code/modules/surgery/organs/lungs.dm]: Bring back this var; still useful.

**Comment by Fox-McCloud** [comment]
Initial thoughts: 

- SR probably need buffed
- Need to tweak poison damage to Vox breathing oxygen

**Comment by AffectedArc07** [comment]
I have played three whole rounds of medical with this system in place and I have made my descision

[Its a no, from me](https://www.youtube.com/watch?v=N5dpeHqWEGM)

File: code/modules/mob/living/carbon/human/species/vox.dm (hunk 4 above)
Comment (Mitchs98) [reply code/modules/mob/living/carbon/human/species/vox.dm]: It's not really strange at all that a slime doesn't have a heart, no.

**Comment by Ozewse** [comment]
> CMO's hypospray starts with mitocholide as opposed to omnizine.

Only thing I 100% agree with out of all the medbay PRs. This is the only SR change we need. 

>pulled back cryo tubes a bit to further prevent them from being cure-alls.

Not without fixing my sleepers first. Saline glucose isn't stabilizing patients fast enough. You want to get rid of the specialized chemicals in it, fine. I can see why. It makes surgeon and MD harder, but I'm good at those roles. I argued for hours in the Discord about how the changes to sleepers were redundant because actually shitty doctors use cryo tubes for everything and don't even know what half the things in a sleeper do. That said, being that the previous PR made all of medbay hinge on chemists- This nerf should be able to be compensated for in game with a good cryomix, right?

>As this is now reagent-medbay, Fix O Vein now does the repair work for internal organs (as opposed to a bruise pack).

Seems like making the surgery kit smaller, might be a good idea? But bruise packs are easy to come by and work for a bajillion surgeries before they run out anyway. Canon - Bruise packs have a few bandaids and hundreds of little sheets of regenerative membrane in them for patching up organs. 

>This massive PR, making changes to how everyone dies -and- multiple medbay systems. 

Should be broken up into multiple entries for clarity.

**Comment by Fox-McCloud** [comment]
re-adds the beakers and droppers to the medical vendors

**Comment by AffectedArc07** [comment]
Things to note after playing even more rounds of this
- The random event that nukes the vendors makes medbay useless since it is so reliant on them and you cannot rebuild them
- Medbay is also far too reliant on the cloner now, like more reliant on the cloner than we used to be on cryotubes
- The lack of auto injectors in boxes is a **REALLY** big issue

**Comment by Spartan6** [comment]
It’s becoming normal now to see multiple cloning tubes. I’ve seen no effective change really, nothing possitive at least, I’m sorry to say.

**Comment by Mitchs98** [comment]
The Changelog and OP needs to be revived  so people have an easier way of seeing what all is changed and do not assume that what's there is the only thing changed. It'll give people an easier way to actually judge the PR as there's a lot of posts on this PR and the changes are spread out/not really well documented.

**Comment by farie82** [comment]
Tried it in action and I don't like it sorry. Just feels... dull... Cloning is the to go to option now. Getting into crit means you can still do stuf but die eventually and can't get back. So short term you can live longer, maybe get out of trouble. But long term you get punished harder. I liked the previous version where you go down when you're almost dead. Last game I played I had to kill all of security who responded to me because they kept standing up with 10 HP left (-90 HP). I dunno it just doesn't make the game better.

**Comment by KasparoVy** [comment]
I've noticed it's much easier for me to close and walk away from the game 🤔 instead of wondering if I have a chance to survive I'm far more certain of my demise.

I've had to employ the tactic of resting and playing dead more regularly because my character gets up and makes people think I'm still a threat (or something to be killed in the case of murderboners)

I've also avoided medical roles like the plague. I haven't had a good time of it recently but now I feel like the atmosphere is too tense to enjoy that sort of thing.

**Comment by farie82** [comment]
Small issues with this as addition. The cardiac arrest thing is a virus and can be made into a culture. Unsure if you can infect others with the culture but hey.

**Comment by Ralta** [comment]
I was very supportive of this initially, but little tweaks and changes along the way has pushed me the other way. As it stands, I don't support the PR in its current state, but feel it would be more viable with the following changes:

- Oxyloss mute should happen MUCH higher than 10. 50+, imo.
- Re-add epi pens to emergency boxes. Most people will die, and be unrevivable in any form whatsoever by the time they reach medbay.
- Defib timer needs to be increased to 5 mins.
- Cloner needs to take a lot longer or require more material.
- Might need some more chems back in sleeper, but difficult to say exactly which would be useful without reverting sleepers to their previous ridiculous state.

Please consider some adjustments back towards viability. I'm all for making medbay a little more interesting/challenging, and making crit more nuanced, but the additional changes feel heavy-handed and are ruining it, in my opinion.

**Comment by AffectedArc07** [comment]
@Ralta 
> Defib timer needs to be increased to 5 mins.

The timer doesnt even exist. Defibs dont rest

**Comment by Ralta** [comment]
> @Ralta
> 
> > Defib timer needs to be increased to 5 mins.
> 
> The timer doesnt even exist. Defibs dont res

Oh you right. Ignore that bit :eyes:

**Comment by Mitchs98** [comment]
> * Cloner needs to take a lot longer or require more material.

I don't exactly agree with this if this is merged. Especially considering how hard it is already to treat newcrit patients. It's nonsensical to make something worse(I.E newcrit) then make something else even worse to compensate(I.E cloning).

If this change does get merged I'm getting up a PR to re-add advanced kits ASAP. Either to medical or as a cargo crate. I have no idea why they were even removed.

**Comment by Ralta** [comment]
> > * Cloner needs to take a lot longer or require more material.
> 
> I don't exactly agree with this if this is merged. Especially considering how hard it is already to treat newcrit patients. It's nonsensical to make something worse(I.E newcrit) then make something else even worse to compensate(I.E cloning).
> 
> If this change does get merged I'm getting up a PR to re-add advanced kits ASAP. Either to medical or as a cargo crate. I have no idea why they were even removed.

The reason cloning should take longer is that if this gets merged, cloning suddenly becomes the easiest solution to most problems. Building extra cloners and getting a ton of material would be simpler than fixing more complex medical problems. I think that would be lessened by some of my other suggestions, but I still feel the point stands.

**Comment by shazbot194** [comment]
> The reason cloning should take longer is that if this gets merged, cloning suddenly becomes the easiest solution to most problems. Building extra cloners and getting a ton of material would be simpler than fixing more complex medical problems. I think that would be lessened by some of my other suggestions, but I still feel the point stands.

Isn't that the problem though? These changes make it so it is much much easier to just clone most people rather then trying to fix them. Would nerfing another system to parody a nerf really be necessary? Shouldn't it be the other way around, and not make the system so harsh so people use it?

**Comment by runemeds** [comment]
Shazbot is right. The concept of nerfing a system that was made stronger by the nerfing of a different system is remarkably bad game design, and a poor choice for game health.

**Comment by Terilia** [comment]
I am mixed about the new system - and here is why. 

I like the idea of reworking crit, of making it more deadly and more serious, but it comes with a huge nerf to systems and makes the overall system regarding medical a two way street. 

Can I fix him fast? - Fix him fast
Anything more complicated? - Into cloning they go. 

This two-way approach to medical is partially due to players being lazy, and partially due to them just not knowing better, the chemist being trash or other unforeseen consequences. 

I had a few rounds where Fox was my Doctor, and I have seen in person that, if you know what you are doing, you can fix people without stuffing them into cloning. In addition, several times he fixed me up from things that I thought were a guaranteed death. Nevertheless, unfortunately not every player is like this. 

Personally, I would ask for more depth in the medical system. I have just taken a short look at TG station, and frankly, I know nothing about how their medical system works. However, the idea alone of brain damage that add random effects sounds far more advanced than what we have right now. I agree, medical should become more interesting, but personally, I would add more systems into the diagram of procedures. Example: Allow them to clone people, but - it might result in brain damage - and woops, person has now "Monophobia - The patient will become increasingly stressed when not in company of others, triggering panic reactions ranging from sickness to heart attacks." which needs to be treated as well. 

In addition: No system should be reliant on one factor. Currently, I feel like the chemist is too important. Destroy the cloner, kill the chemist (or chemist area) – Medbay is done. Back then they still had the defib as a last resort, but this path is now closed. 

Overall - the game is Semi Realistic - allowing us to bend the rules, reality, and time just enough. Because if Survival Games taught me anything: Realism is not always fun.

**Comment by Aurorablade** [comment]
For the phobia/Brain Trauma system we would need some systme ports from TG probaly with thier quirk system. We just need someone to Actually do that..

**Comment by Aurorablade** [comment]
I am seeing some comments about the current Defib system being too damming though, would there be a compromise to that somewhere?

**Comment by AffectedArc07** [comment]
Image
Not 100% sure how related to the PR this is but I am putting it here so it’s seen

**Comment by Aurorablade** [comment]
That color scheme hurts my eyes. ANYWAY, we did have some feedback saying that someone *fox and i am sure anyone else who knows* the system..played..and it actually worked. I really do not think its the system at fault here.

**Comment by Mitchs98** [comment]
> That color scheme hurts my eyes. ANYWAY, we did have some feedback saying that someone _fox and i am sure anyone else who knows_ the system..played..and it actually worked. I really do not think its the system at fault here.

Just because it works doesn't mean it's fit for this server.

**Comment by Raze500** [comment]
Nobody asked for this, but ok.

**Comment by AffectedArc07** [comment]
Why doesnt this have the `[Controversial]` label
